BackofficeBundle/Controller/ContentController.php

```diff
@@ -57,6 +58,21 @@ public function formAction(Request $request, $contentId)
     }
 
     /**
+     * Get the content language from Request or current site
+     * 
+     * @param Request $request
+     * 
+     * @return string
+     */
+    protected function getContentLanguage(Request $request)
+    {
+        if ($request->get('language'))
```

Comment:
or you can simply do $request->get('language', $this->get('open_orchestra.manager.current_site')->getCurrentSiteDefaultLanguage());